.github/workflows/make-release.yml

```diff
@@ -0,0 +1,146 @@
+name: 'Kucher application builder'
+
+# Builds Kucher release
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
I think we should run this (without publishing) on pull requests as well, for testing.

Reply:
Relevant example https://github.com/Zubax/dyshlo/blob/4ae693a1b211b28ae032b19b7374e057e971c379/.github/workflows/verification.yml#L71-L73

Reply:
Using the `github.ref` context in this way is a little risky because it there are other events that match this reference condition. I have kept the events in the `on:` condition.

Reply:
The `on` clause should specify only two events: `push` and `pull_request`, then the risk is eliminated. The point of this change is to run the test for pushes to all branches, not just `master`, as it helps during development. 

Alternatively, we could use `#hashtags` enabling specific workflows to run. We've grown quite used to this kind of automation in other projects as it provides fine control over workflow or even step execution. https://github.com/Zubax/epm_software/blob/3a99f77e364c23d5efdb5fd0c765dd41c3956db1/.github/workflows/main.yml#L10